.vib/vib-platform-verify.json

```diff
@@ -3,39 +3,33 @@
     "package": {
       "context": {
         "resources": {
-          "url": "https://github.com/CartoDB/carto3-helm/tarball/bc22e8f8ece48277a6681ddb2dde37ca8b536ee8",
+          "url": "{SHA_ARCHIVE}",
           "path": "/chart"
         }
       },
       "actions": [
         {
           "action_id": "helm-package"
-        },
-        {
-          "action_id": "linter-packaging",
-          "params": {
-            "kind": "HELM"
-          }
         }
       ]
     },
     "verify": {
       "context": {
         "resources": {
-          "url": "https://github.com/CartoDB/carto3-helm/tarball/bc22e8f8ece48277a6681ddb2dde37ca8b536ee8",
-          "path": "/tests"
+          "url": "{SHA_ARCHIVE}",
+          "path": "/chart"
```

Comment:
I'd switch this to point at the base test directory, which it looks like it's going to be `/.vib/carto/`. Then, in the actions, for example `goss`, you can use the relative path `goss/goss-accounts-www`.